content/docs/user-guide/gems/reference/_index.md

```diff
@@ -172,6 +172,12 @@ toc: true
 | - | - |
 | [In App Purchases](./sdk/in-app-purchases) | The In App Purchases Gem provides functionality for in app purchases in Open 3D Engine projects. |
 
+## Terrain
+
+| Gem | Description |
+| - | - |
+| [Terrain](./terrain) | The Terrain Gem provides tools to create terrain in Open 3D Engine. |
```

Comment:
Here's the description we have in the gem itself, we should probably use the same description here.  Also, instead of creating a new "Terrain" section, this should go in the existing "Environment" section.

"The Terrain Gem is an experimental terrain system.  The terrain system maps height, color, and surface data to regions of the world, provides gradient-based and shape-based authoring tools and workflows, includes specialized rendering for efficient display, and integrates with physics for physical simulation."

Reply:
Done